src/main/service/listParseHelperService.ts

```diff
@@ -189,23 +189,28 @@ export class ListParseHelperService {
      */
     public findAndManipulateJudiciary(session: object): string {
         let judiciaries = '';
```

Comment:
Just like the comment in channel-management. The logic can be simplified so we do not need to keep checking for non-empty string, add delimiter then trim it off later. You could do something like below:

```
let presidingJudiciary = '';
const judiciaries = [];
session['judiciary']?.forEach(judiciary => {
    if (judiciary?.isPresiding === true) {
        presidingJudiciary = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
    } else {
        judiciaries.push(ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs));
    }
});
return [presidingJudiciary, ...judiciaries].filter(j =>  j.trim().length > 0).join(', ');
```